File: Soal-10/PRAK110-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>
#include <math.h>

int main() {
    int alas = 5;
    int tinggi = 12;
    float miring = sqrt(alas*alas + tinggi*tinggi);
    
    printf("\nDiketahui :\n");
    printf("Alas   = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", alas);
    printf("Sisi B = %.0f cm\n", miring);
    printf("Sisi C = %d cm\n", tinggi);
    printf("Keliling = %.0f cm\n", alas + tinggi + miring);
    printf("Luas     = %.0f cm^2\n", (alas * tinggi) / 2.0);
    
    return 0;
}

File: Soal-10/PRAK110-2410817310014-INDRA_SURYADILAGA.py
import math

alas=5
tinggi=12
miring=math.sqrt(alas**2 + tinggi**2)

print(f'''
Diketahui :
Alas   = {alas} cm
Tinggi = {tinggi} cm\n
Jawab :
Sisi A = {alas} cm
Sisi B = {miring:.0f} cm
Sisi C = {tinggi} cm
Keliling = {alas + tinggi + miring:.0f} cm
Luas     = {alas*tinggi/2:.0f} cm^2''')